src/main/java/com/zontik/groshiky/repository/TransactionRepository.java

```diff
@@ -6,4 +6,5 @@
 
 @Repository
 public interface TransactionRepository extends JpaRepository<Transaction, Integer> {
+    void deleteById(Integer id);
```

Comment:
1. "deleteById(ID id)" is already defined in JpaRepository - you don't need to overide it
2. It is not the best solution to totally remove entity from database. Better way is to mark it as not active (for that you need to add boolean field into database entities).

Reply:
In the context of this application, we don't need to keep track of the transaction deletion history. I think, there is no point in storing remote transactions in the database yet.